doc/extension/example_library/front-end/src/ColoredLabel.tsx

```diff
@@ -6,20 +10,29 @@ interface ColoredLabelProps {
   defaultValue?: string;
 }
 
+// Sequence of colors
 const colorWheel = ["#FF0000", "#A0A000", "#00FF00", "#00A0A0", "#0000FF", "#A000A0"]
```

Comment:
could have an array of {color: "#..."}
this way the style returned would be const
It's generally a bad idea to build js object on the fly as they are always different on shallow compare (virtual DOM ...) ...